topics/assembly/tutorials/chloroplast-assembly/tutorial.md

```diff
@@ -131,6 +132,8 @@ Your assembly graph may look like this:
 
 ![assembly graph](../../images/sweet-potato-assembly-graph.png "Assembly graph of the nanopore reads, for the sweet potato chloroplast genome. In your image, some of the labels may be truncated; this is a known bug under investigation.")
 
+*Note: a newer version of the Flye assembly tool now resolves this assembly into a single circle.*
```

Comment:
Could you update the tool ID in the flye step to reflect this?

Reply:
Yes - will change to (toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye/2.6+galaxy0) - so version 2.6, and do you think that's correct with the extra "+galaxy0" part? That's what appears in the tool panel.

Reply:
Yep, that is right.